composer.json

```diff
@@ -8,79 +8,73 @@
-        "symfony/http-client": "^4.4|^5.0",
-        "symfony/http-kernel": "^4.4|^5.0",
-        "symfony/intl": "^4.4|^5.0",
-        "symfony/mailer": "^4.4|^5.0",
-        "symfony/mailgun-mailer": "^4.4|^5.0",
-        "symfony/maker-bundle": "^1.13",
-        "symfony/mime": "^4.4|^5.0",
+        "symfony/form": "^5.0",
+        "symfony/framework-bundle": "^5.0",
+        "symfony/http-client": "^5.0",
+        "symfony/http-kernel": "^5.0",
+        "symfony/intl": "^5.0",
+        "symfony/mailer": "^5.0",
+        "symfony/mailgun-mailer": "^5.0",
+        "symfony/maker-bundle": "^1.14",
+        "symfony/mime": "^5.0",
         "symfony/orm-pack": "^1.0.7",
-        "symfony/security": "^4.4|^5.0",
-        "symfony/security-bundle": "^4.4|^5.0",
-        "symfony/swiftmailer-bundle": "^3.2",
-        "symfony/templating": "^4.4|^5.0",
-        "symfony/translation": "^4.4|^5.0",
-        "symfony/twig-bridge": "^4.4|^5.0",
-        "symfony/twig-bundle": "^4.4|^5.0",
-        "symfony/validator": "^4.4|^5.0",
-        "symfony/web-profiler-bundle": "^4.4|^5.0",
+        "symfony/security-bundle": "^5.0",
+        "symfony/templating": "^5.0",
+        "symfony/translation": "^5.0",
+        "symfony/twig-bridge": "^5.0",
+        "symfony/twig-bundle": "^5.0",
+        "symfony/validator": "^5.0",
+        "symfony/web-profiler-bundle": "^5.0",
         "symfony/webpack-encore-bundle": "^1.6",
-        "symfony/yaml": "^4.4|^5.0",
-        "symplify/autodiscovery": "^6.1",
-        "symplify/flex-loader": "^6.1",
-        "symplify/package-builder": "^6.1",
-        "symplify/statie": "^6.1",
-        "twig/extensions": "^1.5",
-        "vich/uploader-bundle": "^1.10"
+        "symfony/yaml": "^5.0",
+        "symplify/autodiscovery": "^7.1",
+        "symplify/flex-loader": "^7.1",
+        "symplify/package-builder": "^7.1",
+        "symplify/statie": "^7.1",
+        "vich/uploader-bundle": "^1.12"
     },
     "require-dev": {
         "latte/latte": "^2.5",
         "nette/application": "^3.0",
+        "nette/bootstrap": "^3.0",
         "nette/forms": "^3.0",
-        "phpstan/phpstan": "^0.11.16",
-        "phpstan/phpstan-doctrine": "^0.11.6",
-        "phpunit/phpunit": "^8.4",
-        "rector/rector": "^0.5",
-        "roave/security-advisories": "dev-master",
-        "spatie/http-status-check": "^3.1",
-        "symfony/var-dumper": "^4.4|^5.0",
-        "symfony/browser-kit": "^4.4|^5.0",
-        "symplify/coding-standard": "^6.1",
-        "symplify/easy-coding-standard": "^6.1",
-        "symplify/phpstan-extensions": "^6.1"
+        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan-doctrine": "^0.12",
+        "phpunit/phpunit": "^8.5",
+        "rector/rector": "dev-master",
+        "symfony/var-dumper": "^5.0",
+        "symfony/browser-kit": "^5.0",
+        "symplify/coding-standard": "^7.1",
+        "symplify/easy-coding-standard": "^7.1",
+        "symplify/phpstan-extensions": "^7.1"
```

Comment:
Note: `symfony/xxxx` * + config version cannot be used, because of `symfony/error-renderer`, that somehow blocks Symfony 5 install and forces 4.4